_integrations/index.md

```diff
@@ -9,60 +9,102 @@ redirect_from:
+  ![Integrations dashboard with visualizations]({{site.url}}{{site.baseurl}}/images/integrations/nginx-integration-dashboard.png)
 
-## Add integrations
+## Loading custom integration assets
 
-If you have not installed any integrations, you'll be prompted to install them from the Integrations interface. Supported integrations are listed in the **Available** window. 
+To load a custom integration asset, follow these steps: 
+
+1. Download an integration artifact from the [catalog repository](https://github.com/opensearch-project/opensearch-catalog/blob/main/docs/integrations/Release.md).
+2. Go to **Dashboards Management** > **Saved objects**.
+3. Select **Import** on the upper-right toolbar menu, navigate to the folder where you saved the integration artifact, and then choose the file (a file with an .ndjson extension). An example of this step is shown in the following image.
+  ![Import folder window]({{site.url}}{{site.baseurl}}/images/integrations/integration-import-file.png)
+4. Select the saved object you uploaded to confirm that it was uploaded to **Saved objects**. An example of this step is shown in the following image.
+  ![List of integrations saved objects]({{site.url}}{{site.baseurl}}/images/integrations/select-uploaded-integration.png)
+
+---
 
-To add an integration, select the desired prepackaged assets. Currently, OpenSearch Integrations has two flows: Add or Try it. The following example uses the Try it flow:
+## Developer resources
 
-1. On the **Integrations** page, select **NginX Dashboard**.
-2. Select the **Try it** button. _The Try it flow automatically creates a sample index template, adds sample data to the template, and then creates the integration based on that data._
-3. View the asset list and select a dashboard asset.
-4. Preview the data visualizations and sample data details.  
+See the following developer resources for sample code, articles, tutorials, and an API reference:
 
+- [OpenSearch Integrations repository](https://github.com/opensearch-project/opensearch-catalog)
+- [OpenSearch Integrations reference documentation](https://github.com/opensearch-project/opensearch-catalog/tree/main/docs/integrations)
+- [OpenSearch Observability Catalog](https://htmlpreview.github.io/?https://github.com/opensearch-project/opensearch-catalog/blob/main/integrations/observability/catalog.html)
+- [OpenSearch Observability Catalog release page](https://github.com/opensearch-project/opensearch-catalog/blob/main/docs/integrations/Release.md)
+- [Simple Schema for Observability](https://github.com/opensearch-project/opensearch-catalog/tree/main/docs/schema/observability)
+
+---
+
+## Community contribution
+
+The OpenSearch Project seeks your feedback on this feature. Post on the [OpenSearch forum](https://forum.opensearch.org/) to let us know how OpenSearch Integrations works for you or how it can be improved.
```

Comment:
```suggestion
The OpenSearch Project seeks your feedback on this feature. Post on the [OpenSearch forum](https://forum.opensearch.org/) to let us know how **Integrations** works for you or how it can be improved.
```